packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -51,6 +56,34 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  const jsonSchemaConfig:
+    | WorkspaceConfiguration
+    | undefined = workspace.getConfiguration().get("json.schemas");
+  if (
+    process.env["THEIA_PARENT_PID"] !== undefined &&
+    isArray(jsonSchemaConfig)
+  ) {
+    const providedManifestSchemaConfigs = find(jsonSchemaConfig, (_) => {
+      return some(
+        _.fileMatch,
+        (fileMatchEntry) => fileMatchEntry === "manifest.json"
+      );
+    });
+
+    if (providedManifestSchemaConfigs === undefined) {
+      const manifestSchemaConfig = {
+        fileMatch: ["manifest.json"],
+        url:
+          "https://cdn.jsdelivr.net/gh/SAP/ui5-language-assistant/packages/vscode-ui5-language-assistant/resources/manifest-schema/rel-1.19/schema/schema.json",
```

Comment:
Is it available in China?

Reply:
good question, but its the same URL that is currently supposed to be used in China.
This is not a new feature, rather a workaround.